bootstrap.sh

```diff
@@ -1,54 +1,67 @@
-#!/bin/bash
+#!/usr/bin/bash
 
-set -eux pipefail
+set -x  # Exit with nonzero exit code if anything fails
```

Comment:
This [example](https://docs.travis-ci.com/user/job-lifecycle#complex-build-commands) uses `set -ev`, maybe try that for failing CI ?